monster-cards/src/stores/cardStore.ts

```diff
@@ -12,74 +15,68 @@ export interface Card {
+	{ id: 'card-mayhem-10', cardType: 'mayhem', name: 'Scope Creep', description: 'Apparently, a "few small changes" means 17 new user stories.', score: 3, tagline: '', imageClass: 'Cards-Mayhem10', favouriteCount: 0 },
+	{ id: 'card-mayhem-11', cardType: 'mayhem', name: 'Slingshot', description: 'DevOps changed the authentication scheme but how it affects your project is "not their problem".', score: 2, tagline: '', imageClass: 'Cards-Mayhem11', favouriteCount: 0 },
+	{ id: 'card-mayhem-12', cardType: 'mayhem', name: 'The Eraser', description: 'The only thing complete about the UI design is how completely lacking it is in detail.', score: 1, tagline: '', imageClass: 'Cards-Mayhem12', favouriteCount: 0 },
+	{ id: 'card-mayhem-13', cardType: 'mayhem', name: 'The Gambler', description: 'The closer the deadline gets, the more liberal your team becomes with the definition of "done".', score: 1, tagline: '', imageClass: 'Cards-Mayhem13', favouriteCount: 0 },
+	{ id: 'card-mayhem-14', cardType: 'mayhem', name: 'The Horde', description: 'Instead of more time to meet your requirements, you got a dozen untrained interns to "help".', score: 3, tagline: '', imageClass: 'Cards-Mayhem14', favouriteCount: 0 },
+	{ id: 'card-mayhem-15', cardType: 'mayhem', name: 'The Inventor', description: 'When discussing what framework to use, an enthusiastic voice says, "We\'ll just roll our own!"', score: 1, tagline: '', imageClass: 'Cards-Mayhem15', favouriteCount: 0 },
+	{ id: 'card-mayhem-16', cardType: 'mayhem', name: 'The Inviter', description: 'You and your team have been invited to daily meetings to discuss your "lack of progress".', score: 4, tagline: '', imageClass: 'Cards-Mayhem16', favouriteCount: 0 },
+	{ id: 'card-mayhem-17', cardType: 'mayhem', name: 'The Juggler', description: 'Flimsy dates, changing priorities and bad assumptions are the pillars of your project manager\'s master plan!', score: 1, tagline: '', imageClass: 'Cards-Mayhem17', favouriteCount: 0 },
+	{ id: 'card-mayhem-18', cardType: 'mayhem', name: 'Sigh-Low', description: 'While on a conference call it becomes painfully clear...two different teams just completed the same work.', score: 2, tagline: '', imageClass: 'Cards-Mayhem18', favouriteCount: 0 },
+	{ id: 'card-mayhem-19', cardType: 'mayhem', name: 'Caveman Coder', description: 'You brought in a contractor, who smashed your code until it "worked" and then he committed it to master.', score: 2, tagline: '', imageClass: 'Cards-Mayhem19', favouriteCount: 0 }
+];
 
-export function bindActions() {
-	return cardStore.observe().subscribe((options: any) => {
-			const changeRecord = <ChangeRecord> options;
+const cardStore = cardStoreFactory();
 
-			return putCard.do(changeRecord);
+export function bindActions() {
```

Comment:
I was trying to follow the original logic by using observable as shown below. But it didn't work. @maier49 , any comments will be highly appreciated!
```typescript
	const subscription = cardStore.observe().subscribe((options) => {
			return putCard.do(options);
	});
	cardStore.add(cards);
	return subscription;
```